### PR TITLE
Added build number to DLL

### PR DESCRIPTION
**Pull request by alexey-tatarenko**
Added build number to DLL

### CODE REVIEW

File: C/7zVersion.h [hunk 1]
```diff
@@ -1,6 +1,6 @@
 #define MY_VER_MAJOR 24
 #define MY_VER_MINOR 07
-#define MY_VER_BUILD 0
+#define MY_VER_BUILD GithubBuildNumber
```
Comment (MaksymSlota) [inline C/7zVersion.h]: lets add 
`#define MY_VER_PATCH GithubBuildNumber`
and use it in rc file
`#define MY_VER MY_VER_MAJOR,MY_VER_MINOR,MY_VER_BUILD,MY_VER_PATCH`

Comment (alexey-tatarenko) [reply C/7zVersion.h]: OK.

File: C/7zVersion.rc [hunk 2]
```diff
@@ -11,7 +11,7 @@
 #include "7zVersion.h"
 #endif
 
-#define MY_VER MY_VER_MAJOR,MY_VER_MINOR,MY_VER_BUILD,0
```
Comment (MaksymSlota) [inline C/7zVersion.rc:14]: `#define MY_VER MY_VER_MAJOR,MY_VER_MINOR,MY_VER_BUILD,MY_VER_PATCH`

Comment (alexey-tatarenko) [reply C/7zVersion.rc:14]: Done.

**Review by MaksymSlota — APPROVED** [review]

**Pull request merged**